edxadapt/edxadapt.py

```diff
@@ -60,7 +60,27 @@ class EdxAdaptXBlock(StudioEditableXBlockMixin, XBlock):
         scope=Scope.preferences
     )
 
-    editable_fields = ('params', 'skills', 'edx_adapt_api_url')
+    success_registration_msg = String(
+        default=_(
+            "Your adaptive learning session is configured now. Please click Next button to start solve problems."
+        ),
+        display_name=_("Successful EdxAdapt registration message"),
+        help=_("Message displayed to student after successful registration in EdxAdapt."),
+        scope=Scope.content,
+    )
+
+    fail_registration_msg = String(
+        default=_(
+            "There was a technical issue while we were configuring your adaptive learning session. Please try to "
+            "refresh this page or contact technical staff if problem persists."
+        ),
+        display_name=_("Failed EdxAdapt registration message"),
+        help=_("Assistance message with steps student can do to improve Edx Adapt enrollment, e.g. email could be "
+               "added for asking for technical support."),
+        scope=Scope.content,
+    )
+
+    editable_fields = ('params', 'skills', 'edx_adapt_api_url', 'success_registration_msg', 'fail_registration_msg')
```

Comment:
All line's length is less than 120 chars which is recommended by edx, so I suppose splitting is not needed.